KsApi/models/BackingTests.swift

```diff
@@ -14,7 +14,7 @@ final class BackingTests: XCTestCase {
         "expiration_date": "2019-09-23",
         "id": 20,
         "last_four": "1234",
-        "payment_type": "CREDIT_CARD",
+        "payment_type": "APPLE_PAY",
```

Comment:
Is it necessary to change this?

Reply:
Not really, I had changed it just to make sure everything was working. I will revert it 👍